cfgov/scripts/initial_data.py

```diff
@@ -9,66 +9,65 @@
 
 
 def run():
-    if settings.DEBUG:
-        print 'Running script \'scripts.initial_data\' ...'
-        admin_user = None
-        site_root = None
-        events = None
+    print 'Running script \'scripts.initial_data\' ...'
```

Comment:
Can we `from __future__ import print_function` and use `print()` here, since we're changing it?